jupyterhub/apihandlers/users.py

```diff
@@ -220,11 +220,11 @@ def post(self, name):
         user = self.find_user(name)
         if user is None:
             raise HTTPError(404, "No such user %r" % name)
-        if user.running:
-            # include notify, so that a server that died is noticed immediately
-            state = yield user.spawner.poll_and_notify()
-            if state is None:
-                raise web.HTTPError(400, "%s's server is already running" % name)
+        #if user.running:
```

Comment:
@minrk  am I right that `user.running` is expected to be the only server running?
Same goes for `self.spawn_pending` and `self.stop_pending`

So we should change them in something like:

```python
self.spawners[server_name].running
self.spawners[server_name].spawn_pending
self.spawners[server_name].stop_pending
```